### PR TITLE
feat: automl ga samples

### PR DESCRIPTION
**Pull request by sirtorry**
todo:
- tests
- readme
- etc.

### CODE REVIEW

**Comment by czahedi** [comment]
This is apparently passing the checks. Can we merge these to master?

**Comment by sirtorry** [comment]
that's because i haven't added any tests yet 🙃

File: automl/src/get_model_evaluation.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    printf('seconds: %d' . PHP_EOL, $modelEvaluation->getCreateTime()->getSeconds());
+    printf('nanos : %d' . PHP_EOL, $modelEvaluation->getCreateTime()->getNanos());
+    printf('Evaluation example count: %s' . PHP_EOL, $modelEvaluation->getEvaluatedExampleCount());
+    // [END automl_language_sentiment_analysis_get_model_evaluation]
+    // [END automl_language_text_classification_get_model_evaluation]
+    // [END automl_translate_get_model_evaluation]
+    // [END automl_vision_classification_get_model_evaluation]
+    // [END automl_vision_object_detection_get_model_evaluation]
+    printf('Model evaluation metrics: %s' . PHP_EOL, $modelEvaluation->getTextExtractionEvaluationMetrics());
+    // [END automl_language_entity_extraction_get_model_evaluation]
```
Comment (bshaffer) [inline automl/src/get_model_evaluation.php:67]: These tags are duplicated and start/end in the wrong places!

**Comment by sirtorry** [comment]
Deleting branch. Does not exist in docs today.

**Pull request closed without merging**